testdata/scripts/convert_comment.txt

```diff
@@ -0,0 +1,22 @@
+env HOME=$WORK/home
+
+gunk convert util.proto
+cmp util.gunk util.gunk.golden
+
+-- util.proto --
```

Comment:
this test script should also test multi-line doc comments.

Reply:
it seems that we currently doesn't handle the form `/* ... */`. We always transform to `//` form. Should I raise new issue?

Reply:
No, I don't think that's necessary.